src/main/java/dk/kb/ds/cumulus/export/CumulusExport.java

```diff
@@ -1,7 +1,79 @@
 package dk.kb.ds.cumulus.export;
 
+import dk.kb.cumulus.CumulusQuery;
+import dk.kb.cumulus.CumulusRecord;
+import dk.kb.cumulus.CumulusRecordCollection;
+import dk.kb.cumulus.CumulusServer;
+import dk.kb.cumulus.utils.ArgumentCheck;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
 public class CumulusExport {
-    public static void main(String[] args) {
-        System.out.println("Cumulus exporter");
+    public static void main(String[] args) throws IOException, TransformerException, ParserConfigurationException {
+
+        try (CumulusServer server = new CumulusServer(Configuration.getCumulusConf())) {
+            String myCatalog = Configuration.getCumulusConf().getCatalogs().get(0);
+            CumulusQuery query = CumulusQuery.getQueryForAllInCatalog(myCatalog);
+            CumulusRecordCollection recordCollection = server.getItems(myCatalog, query);
+
+            File outputFile = new File("solrInputFile.xml");
+            OutputStream out = new FileOutputStream(outputFile);
+
+            ArgumentCheck.checkNotNull(out, "OutputStream out");
+            DocumentBuilderFactory docFactory = DocumentBuilderFactory.newInstance();
+            DocumentBuilder docBuilder = docFactory.newDocumentBuilder();
+            Document document = docBuilder.newDocument();
+            Element rootElement = document.createElement("add");
+            document.appendChild(rootElement);
+
+            for (CumulusRecord record : recordCollection) {
+                Element docElement = document.createElement("doc");
+                rootElement.appendChild(docElement);
+                try {
+                    String id = record.getFieldValueOrNull("guid");
+                    String titleC = record.getFieldValueOrNull("Titel");
+                    String dateC = record. getFieldValueForNonStringField("Date Time Digitized");
```

Comment:
Solr timestamp parsing is very picky. We're trying to use a [DateRangeField](https://lucene.apache.org/solr/guide/8_1/working-with-dates.html#date-range-formatting), which means variable precision, but it still needs to be in ISO date format and Zulu-time (also known as UTC time), e.g. `2019-10-02T12:41:00Z`. A Cumulus-time such as `Tue Feb 27 14:15:02 CET 2018` must be converted to `2018-02-27T12:15:02Z`.

I recommend using either Oracle's [DateTimeFormatter](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html) or Joda-Time's [DateTimeFormatter](https://www.joda.org/joda-time/apidocs/org/joda/time/format/DateTimeFormat.html). Joda-Time it a bit smarter for datetime-handling, but if we only it for the current situation then it is probably overkill and we should stick to the standard Oracle one.

Reply:
Pushed changes,

Reply:
It _looks_ right, but I am not all that confident in my own date.handling abilities. Could you add a unit-test for the `getUTCTime`-method so that we can check that it handles the timezone-magic properly?

Reply:
Added unit test